hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanActionExecutor.java

```diff
@@ -223,6 +223,10 @@ private HoodieCleanMetadata runClean(HoodieTable<T, I, K, O> table, HoodieInstan
       );
       if (!skipLocking) {
         this.txnManager.beginTransaction(Option.of(inflightInstant), Option.empty());
+        table.getMetaClient().reloadActiveTimeline();
```

Comment:
already we do reload the timeline at L218. So, do we still need this reload? 
if we are concerned about TOCTOU, even there could be a gap here right. just after L 228, the clean could have been completed by another concurrent writer.

Reply:
When we are inside the lock it can't right?